modules/ROOT/pages/_partials/keycloak/binding.adoc

```diff
@@ -0,0 +1,11 @@
+:service-name: Identity Management
```

Comment:
@jstaffor do you want to edit keycloak stuff in this pr as well?

Reply:
oh i get it, this is from my quick fix.

Quick update: ak-47 broke "getting started" build so I had to manually reintroduce these two files (keycloak/binding.adoc, /keycloak/keycloak-setup.adoc)

Reply:
i think if you will make a pr push_overhaul->master, these will not be included, so yeah, no worries